frontend/appflowy_flutter/integration_test/document_test.dart

```diff
@@ -25,6 +27,40 @@ void main() {
       await TestFolder.cleanTestLocation(null);
     });
 
+    testWidgets(
+        'change the language successfully when launching the app for the first time',
+        (tester) async {
+      await tester.initializeAppFlowy();
+
+      await tester.tapLanguageSelectorOnWelcomePage();
+      expect(find.byType(LanguageItemsListView), findsOneWidget);
+
+      await tester
+          .tapLanguageItem(languageCode: 'zh', countryCode: 'CN')
+          .whenComplete(
+        () {
+          expect(
+            find.byWidgetPredicate(
+              (widget) => widget is FlowyTextButton && widget.text == '开始',
+            ),
+            findsOneWidget,
+          );
+        },
+      );
+
+      await tester
+          .tapLanguageItem(languageCode: 'en', scrollDelta: -100)
+          .whenComplete(() {
+        expect(
+          find.byWidgetPredicate(
+            (widget) =>
+                widget is FlowyTextButton && widget.text == 'Quick Start',
+          ),
+          findsOneWidget,
+        );
+      });
+    });
+
```

Comment:
move this test to a new file.

Reply:
There is one more missing test: we should verify that the language remains the same as before after relaunching.